lib/models/project.js

```diff
@@ -96,6 +97,8 @@ class Project {
     if (this.packageInfoCache.hasErrors()) {
       this.packageInfoCache.showErrors();
     }
+
+    this.singletonAddonCache = new SingletonAddonCache(this);
```

Comment:
This should be underscored:

```suggestion
    this._singletonAddonCache = new SingletonAddonCache(this);
```

Reply:
I'll change the name to _perBundleAddonCache, but otherwise agree.